datacontract/export/duckdb_type_converter.py

```diff
@@ -48,6 +48,9 @@ def convert_to_duckdb_json_type(field: Field) -> None | str:
     if datacontract_type.lower() in ["array"]:
         return convert_to_duckdb_json_type(field.items) + "[]"  # type: ignore
     if datacontract_type.lower() in ["object", "record", "struct"]:
+        # If no fields are defined, treat as generic JSON
+        if field.fields is None or len(field.fields) == 0:
+            return "JSON"
```

Comment:
@simonharrer what do you think? How should an object without fields be treated?

Reply:
Two options
- introduce a general fallback to `text`
- use `JSON` for this particular case

I think both is fine. The solution will probably always be to use the `physicalType` to override this.